test/unit/controllers/declaration/FiscalInformationControllerSpec.scala

```diff
@@ -99,6 +99,18 @@ class FiscalInformationControllerSpec extends ControllerSpec with OptionValues {
         theResponseForm.value.value.onwardSupplyRelief mustBe yes
       }
 
+      "mode is Change and there is additionalFiscalReferencesData in cache" in {
```

Comment:
Nit: It is enough to say "mode is Change". This way it is clear that it is not dependent on any other variable.